packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -1382,8 +1388,13 @@ class _ContextMenuSheet extends StatelessWidget {
   final List<Widget> actions;
   final _ContextMenuLocation _contextMenuLocation;
   final Orientation _orientation;
+  // Link the scrollbar to the scrollview by providing them the same scroll
+  // controller.
+  final ScrollController _controller = ScrollController();
```

Comment:
This scroll controller is not disposed?

Reply:
Thanks for the heads up.

Reply:
Wait actually @navaronbracke this class is a stateless widget, would we still need to dispose the controller, and if so, how?

Reply:
Hmm, in that case @polina-c what is the guidance here? Since the main reason why I asked if this could be done is for leak tracking purposes (otherwise we'll end up with tests that fail with leak tracking enabled?) Do we convert it to a stateful widget?

Reply:
Thanks!
Yes, if a controller is created by a widget, it should be stateful widget.